### ISSUE TITLE
Does rust sgx sdk support generating public key certificates in x509 format?

### ISSUE THREAD

**Issue by HGZ-20** [apache/teaclave-sgx-sdk#442]
I want to generate a public key certificate in the enclave and send it out, but I don't seem to find a way to generate a public key certificate in x509 format in the rust sgx sdk. Only see the `get_ecc_cert()` method written in SampleCode. Is it because exporting certificates is not supported yet, and I have to write related codes by myself?

**Comment by yangfh2004** [comment]
Yes, you do, if you want to use it with secured connections, you need a CA to sign your pub key otherwise you can sign it by yourself inside the enclave. You can generate a pair of RSA keys with the SDK but you need to do the rest.
